packages/create-bison-app/template/_templates/graphql/new/graphql.ejs

```diff
@@ -25,21 +33,32 @@ export const <%= camelized %>Queries = extendType({
   type: 'Query',
   definition: (t) => {
     // List <%= plural %> Query
-    t.list.field('<%= plural.toLowerCase() %>', {
-      type: '<%= camelized %>',
+    t.field('<%= plural.toLowerCase() %>', {
+      type: list('<%= camelized %>'),
```

Comment:
correct me if I'm wrong but I think both of these are equivalent.  I feel `t.list.field` is slightly more straightforward due to the intellisense autocompletion and not requiring an additional import of the list function. thoughts?

Reply:
We made this change when troubleshooting the other day. 
It is the same... 

I'll revert and save an import. 👍🏼